### PR TITLE
Added support for examples for VertexAI chat models.

### PR DESCRIPTION
**Pull request by lkuligin**
#5278

### CODE REVIEW

**Comment by bitomukesh** [comment]
@codiumAI-Agent

File: langchain/chat_models/vertexai.py [hunk 1]
```diff
@@ -120,13 +150,16 @@ def _generate(
...
+        if examples:
+            params["examples"] = _parse_examples(examples)
         if not self.is_codey_model:
             chat = self.client.start_chat(
                 context=context, message_history=history.history, **params
             )
         else:
             chat = self.client.start_chat(**params)
-        response = chat.send_message(question.content, **params)
+        response = chat.send_message(question.content)
```
Comment (baskaryan) [inline langchain/chat_models/vertexai.py:162]: is this a backwards compatible change?

Comment (lkuligin) [reply langchain/chat_models/vertexai.py:162]: yes, since we send the same params when we start the `chat` and we don't persist the ChatSession.

Comment (baskaryan) [reply langchain/chat_models/vertexai.py:162]: nice!

**Pull request merged**